services/src/test/java/org/keycloak/vault/PlainTextVaultProviderFactoryTest.java

```diff
@@ -70,6 +91,1258 @@ public void shouldReturnNullWhenWithNullDirectory() {
     }
 
     /**
+     * A whitebox implementation of the Realm model, which is needed to extract realm name.
+     * Please use only for testing {@link FilesPlainTextVaultProviderFactory}.
+     */
+    private static class VaultRealmModel implements RealmModel {
```

Comment:
Remove changes to RealmModel as this is not needed, then revert this class as well